orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf

```diff
@@ -12,14 +12,20 @@
 ################################################################################
 
 module "orc8r" {
-  # Change this to pull from github with a specified ref
+  # Change this to pull from GitHub with a specified ref, e.g.
+  # source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref=v1.6"
   source = "../../../orc8r-aws"
 
   region = "us-west-2"
 
-  orc8r_db_password           = "mypassword" # must be at least 8 characters
+  # If you are performing an upgrade, and are targeting a Postgres version
+  # earlier than current default (12), you must pin to your current version
+  # orc8r_db_engine_version     = "9.6"
```

Comment:
can we change the engine_version to `9.6.15` for consistency?

Reply:
xposting from above: no longer relevant